### DISCUSSION TITLE
How to make people sponsor you on github ?

### DISCUSSION THREAD

**Discussion by DhanushNehru** [community/community#18438] [Other Feature Feedback, Questions, & Ideas]
Inspite of contributing to projects and coding regularly there are still cases in which developers dont even get a single sponsor.

Can anyone provide a roadmap or method in order to receive sponsors for a sponsor enabled github profile ?

**Comment by proarafat** [comment]
Learn More from here: https://github.com/sponsors
there have more details about sponsar

**Reply by Sulagna-Dutta-Roy** [reply]
@DhanushNehru I can refer you this blog: https://calebporzio.com/i-just-hit-dollar-100000yr-on-github-sponsors-heres-how-i-did-it
